web/auth/oauth.go

```diff
@@ -554,7 +579,7 @@ func authorizeSharing(c echo.Context) error {
 	return c.Redirect(http.StatusSeeOther, redirect.String())
 }
 
-func cancelAuthorizeSharing(c echo.Context) error {
+func (a *AuthHTTPHandler) cancelAuthorizeSharing(c echo.Context) error {
```

Comment:
I find the comments by Codecov a bit annoying

Reply:
Yep, agree. I will try to remove them.